pkg/cli/cliexecuter.go

```diff
@@ -127,7 +127,7 @@ func (e *Executer) runWithRetry(ctx context.Context, uid types.UID, command []st
 			ctxWithTimeout, cancel := context.WithTimeout(ctx, timeout)
 			defer cancel()
 			stdout, stderr, faErr = e.runner(ctxWithTimeout, command)
-			if faErr == nil {
+			if faErr == nil && stderr == "" {
```

Comment:
stderr stream isn't an indication for an actual error!
e.g. curl uses stderr for verbose info, and stdout for the actual result

/hold